opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -646,7 +646,7 @@ message IntHistogramDataPoint {
   //
   // (-infinity, explicit_bounds[i]] for i == 0
   // (explicit_bounds[i-1], explicit_bounds[i]] for 0 < i < N-1
-  // (explicit_bounds[i], +infinity) for i == N-1
+  // (explicit_bounds[i-1], +infinity) for i == N-1
```

Comment:
I think this is wrong. Lets assume only one bound [1.0]:

(-infinity, explicit_bounds[i]] for i == 0 -> (-infinity, 5.0]
(explicit_bounds[i-1], +infinity) for i == N-1 (segfault, infinity) (segfault because N=1 then N-1=0 then i=-1

Reply:
May want to use the values for i where i is fixed:

```
  // (-infinity, explicit_bounds[0]]
  // (explicit_bounds[i-1], explicit_bounds[i]] for 0 < i < N-1
  // (explicit_bounds[N-1], +infinity)
```